### PR TITLE
add sale product

### PR DESCRIPTION
**Pull request by nguyenhonghai1997bg**

### CODE REVIEW

**Comment by nguyenhonghai1997bg** [comment]
ready

File: project1/app/Http/Controllers/Admin/SaleController.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $data = [];
+        $data['price'] = $request->get('priceSale');
+        $data['content'] = $request->get('contentSale');
+        $data['product_id'] = $request->get('product_id');
+        $sale = $this->model->create($data);
```
Comment (BunnyPi04) [inline project1/app/Http/Controllers/Admin/SaleController.php:52]: đưa vào repository

File: project1/routes/web.php [hunk 2]
```diff
@@ -111,6 +112,7 @@
     Route::post('enter-info', 'OrderDetailController@addOrder')->name('add-order');
     Route::post('user-search-products-by-price', 'ProductController@searchByPrice')->name('user-search-products-by-price');
     Route::get('searchWithCategory/{id}', 'ProductController@searchWithCategory')->name('searchWithCategory');
+    Route::get('detail-order/{id}', 'Admin\DetailOrderController@userShow')->name('userdetail.order')->middleware('auth');
```
Comment (BunnyPi04) [inline project1/routes/web.php]: Tất cả name route đặt snake_case with dot notation nhé
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_12-dat-ten-trong-laravel-4

**Comment by BunnyPi04** [comment]
commented

File: project1/app/Http/Controllers/Admin/SaleController.php (hunk 1 above)
Comment (nguyenhonghai1997bg) [reply project1/app/Http/Controllers/Admin/SaleController.php:52]: chỗ ý là em gọi trong repository ra mà chị ơi!

**Comment by nguyenhonghai1997bg** [comment]
ready

File: project1/routes/web.php [hunk 3]
```diff
@@ -111,6 +112,7 @@
     Route::post('enter-info', 'OrderDetailController@addOrder')->name('add-order');
     Route::post('user-search-products-by-price', 'ProductController@searchByPrice')->name('user-search-products-by-price');
     Route::get('searchWithCategory/{id}', 'ProductController@searchWithCategory')->name('searchWithCategory');
```
Comment (BunnyPi04) [inline project1/routes/web.php]: route name đặt lại nhé
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_12-dat-ten-trong-laravel-4

**Comment by BunnyPi04** [comment]
commented

**Comment by nguyenhonghai1997bg** [comment]
ready

**Pull request merged**